host/3.0/buster/amd64/dotnet/dotnet-inproc/start_deprecated.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+echo -e "**WARNING**: You are using an outdated version of the Azure Functions runtime:
+https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4
+===================================================================================="
```

Comment:
may be we can keep only this much in a new script

Reply:
CMD is basically the entrypoint of the script. It runs when the container is started and therefore we have to start the host at the end of any startup script

Reply:
yes i understood this is sort of entrypoint for the container, ignore this